lib/Checkout/Payments/PaymentRecipient.php

```diff
@@ -14,6 +16,11 @@ class PaymentRecipient
      */
     public $account_number;
 
+    /**
+     * @var Address
+     */
+    public $address;
+
     /**
      * @var string
      */
```

Comment:
`Zip` is deprecated, maybe we should add a comment here

Reply:
It's deprecated for NAS but not for MBC, that's why I didn't add the deprecated annotation